src/workerd/api/wpt/dom/abort-test.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+import { type TestRunnerConfig } from 'wpt:harness';
+
+export default {
+  'abort-signal-any.any.js': {
+    includeFile: 'abort-signal-any-tests.js',
+    comment:
+      '(1, 2) Target should be set to signal. (3) Should be investigated.',
+    expectedFailures: [
+      'AbortSignal.any() follows a single signal (using AbortController)',
+      'AbortSignal.any() follows multiple signals (using AbortController)',
+      'Abort events for AbortSignal.any() signals fire in the right order (using AbortController)',
```

Comment:
Quick link to the relevant bit for the "signals fire in the right order" test:

https://github.com/web-platform-tests/wpt/blob/d46d9abdea3ea1eb0f1eebdc4430f21a13afaae3/dom/abort/resources/abort-signal-any-tests.js#L164

Reply:
@jasnell I'm marking this as resolved for now to prepare for PR merge, but here were the failing cases we talked about last time.